inlong-sort/sort-connectors/jdbc/src/main/java/org/apache/inlong/sort/jdbc/internal/JdbcMultiBatchingOutputFormat.java

```diff
@@ -532,6 +515,20 @@ protected void attemptFlush() throws IOException {
         }
     }
 
+    /**
+     * Output metrics with estimate for pg or other type jdbc connectors.
+     */
+    private void outputMetrics(String tableIdentifier, Long rowSize, Long dataSize) {
+        String[] fieldArray = tableIdentifier.split("\\.");
+        if (fieldArray.length == 3) {
+            sinkMetricData.outputMetricsWithEstimate(fieldArray[0], fieldArray[1], fieldArray[2],
+                    false, rowSize, dataSize);
+        } else if (fieldArray.length == 2) {
+            sinkMetricData.outputMetricsWithEstimate(fieldArray[0], null, fieldArray[1],
+                    false, rowSize, dataSize);
+        }
```

Comment:
What are the forms of the string before cutting? Can you give some examples?
In addition, if the array length is neither 2 nor 3, should you print logs to record data information?

Reply:
Executed too many times here. Has log elsewhere

Reply:
I didn't understand where the log was printed? If the length of the table name string after splitting is not as expected, printing the log is very helpful for locating the problem.